packages/component-library/src/Scatterplot/Scatterplot.js

```diff
@@ -228,12 +128,12 @@ const Scatterplot = ({
         </VictoryPortal>
         <VictoryScatter
           animate={{ onEnter: { duration: 500 } }}
-          categories={{ x: dataKeyLabel }}
+//        categories={{ x: categoryData }}
```

Comment:
No more categories?

Reply:
`categories` actually doesn't do anything for the VictoryScatter - it actually gives you an invalid prop warning when you pass it in - but it is a common prop across the other chart components.

Reply:
Have it commented out here because for the other components we will want it.